argoverse/visualization/visualize_sequences.py

```diff
@@ -41,7 +44,10 @@ def viz_sequence(
 
     if lane_centerlines is None:
         # Get API for Argo Dataset map
-        avm = ArgoverseMap()
+        if map_root is not None:
+            avm = ArgoverseMap(map_root)
```

Comment:
What's the default in `ArgoverseMap` if `map_root` isn't passed?  I would have expected it to just be `None` anyway.  If that's the case, you could just do:

```py
avm = ArgoverseMap(map_root)
```

Either way...

Reply:
If you do not pass the `map_root` to `ArogverseMap`, it would be `ROOT=Path(__file__).resolve().parent.parent.parent / "map_files"`. 

Related Links:
- https://github.com/argoai/argoverse-api/blob/master/argoverse/map_representation/map_api.py#L41
- https://github.com/argoai/argoverse-api/blob/master/argoverse/map_representation/map_api.py#L58